x-pack/plugins/enterprise_search/public/applications/app_search/components/app_search_gate/app_search_gate.tsx

```diff
@@ -0,0 +1,628 @@
+            <EuiFlexItem>
+              <EuiText color="subdued" size="xs">
+                <FormattedMessage
+                  id="xpack.enterpriseSearch.workplaceSearch.gateForm.additionalFeedback.description"
+                  defaultMessage=" By submitting feedback you acknowledge that you've read and agree to our {termsOfService}, and that Elastic may {contact} about our related products and services,
+                      using the details you provide above. See {privacyStatementLink} for more
+                      details or to opt-out at any time."
+                  values={{
+                    contact: (
+                      <EuiLink href={docLinks.workplaceSearchGatedFormDataUse}>
+                        <FormattedMessage
+                          id="xpack.enterpriseSearch.workplaceSearch.gateForm.additionalFeedback.contact"
+                          defaultMessage="contact you"
+                        />
+                      </EuiLink>
+                    ),
+                    privacyStatementLink: (
+                      <EuiLink href={docLinks.workplaceSearchGatedFormPrivacyStatement}>
+                        <FormattedMessage
+                          id="xpack.enterpriseSearch.workplaceSearch.gateForm.additionalFeedback.readDataPrivacyStatementLink"
+                          defaultMessage="Elastic’s Privacy Statement"
+                        />
+                      </EuiLink>
+                    ),
+                    termsOfService: (
+                      <EuiLink href={docLinks.workplaceSearchGatedFormTermsOfService}>
+                        <FormattedMessage
+                          id="xpack.enterpriseSearch.workplaceSearch.gateForm.additionalFeedback.readTermsOfService"
+                          defaultMessage="Terms of Service"
+                        />
+                      </EuiLink>
+                    ),
+                  }}
+                />
+              </EuiText>
+            </EuiFlexItem>
+          </EuiFlexGroup>
+        </EuiFormRow>
+
+        <EuiSpacer />
+
+        <EuiFormRow
+          labelAppend={i18n.translate(
+            'xpack.enterpriseSearch.workplaceSearch.gateForm.participateUxLab.optionalLabel',
+            {
+              defaultMessage: 'Optional',
+            }
+          )}
+          label={i18n.translate(
+            'xpack.enterpriseSearch.workplaceSearch.gateForm.participateUxLab.Label',
+            {
+              defaultMessage: 'Join our user research studies to improve Elasticsearch?',
+            }
+          )}
+        >
+          <EuiSelect
+            hasNoInitialSelection
+            options={[
+              {
+                text: i18n.translate(
+                  'xpack.enterpriseSearch.workplaceSearch.gateForm.participateUxLab.Label.Yes',
+                  {
+                    defaultMessage: 'Yes',
+                  }
+                ),
+                value: participateInUXLabsChoice.yes.choice,
```

Comment:
I'm not sure whey this `participateInUXLabsChoice` is needed vs just using `true` & `false` as values. What am I missing here?

Reply:
We want the telemetry to read "yes" or "no" - this is the equivalent to what we did for Workplace Search